shared/test/router.config.test.ts

```diff
@@ -18,6 +22,14 @@ describe('Router Config Builder', () => {
       url: 'https://wg-federation-demo-accounts.fly.dev/graphql',
       subscriptionUrl: '',
       subscriptionProtocol: 'ws',
+      schema: buildSchema(
+        fs.readFileSync(path.join(__dirname, 'testdata', 'subgraphConfigs', 'normalized-accounts.graphql')).toString(),
+      ),
+      configurationDataMap: new Map<string, ConfigurationData>(JSON.parse(
+        fs.readFileSync(path.join(__dirname, 'testdata', 'subgraphConfigs', 'accounts-configuration-data-map.json'))
```

Comment:
Can we not just generate those artifacts right here with the library? Maintaining them is error prone and manual.